src/components/to-do-list/lists/ListItem.js

```diff
@@ -2,9 +2,15 @@ import React from "react";
 import './list.css';
 
 function ListItem(props) {
+
+    function onDeleteItem() {
+        props.deleteItem(props.id);
+    }
+
     return (
-        <li>
-            {props.item}
+        <li className={'listItem'}>
+            {props.text}
+            <button onClick={onDeleteItem} className='btn_delete'>Delete</button>
```

Comment:
put it here like this:
```suggestion
            <button onClick={() => props.deleteItem(props.id)} className='btn_delete'>Delete</button>
```